runtime/browser/runtime_platform_util_linux.cc

```diff
@@ -46,7 +46,8 @@ void XDGOpen(const std::string& path) {
   GURL url(path);
   if (url.is_valid() && url.SchemeIsHTTPOrHTTPS()) {
     std::string cmd = "xdg-open " + path;
-    std::system(cmd.c_str());
+    if (std::system(cmd.c_str()) != 0)
```

Comment:
Maybe it is not this PR problem. Any better option than `std::system()` ? @rakuco